posthog/clickhouse/migrations/0179_add_materialized_columns.py

```diff
@@ -0,0 +1,67 @@
+from posthog.clickhouse.client.connection import NodeRole
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+
+
+def generate_add_columns_sharded_events() -> str:
+    """Generate ALTER TABLE statement for sharded_events with all 40 columns."""
+    statements = []
+
+    # Add string columns (0-9)
+    for i in range(10):
+        statements.append(f"ADD COLUMN IF NOT EXISTS `mat_string_{i}` Nullable(String)")
```

Comment:
naming: how about calling it `dmat_string_0` (dynamic materialised) so the naming doesn't clash with a property called `string_0` being materialized with the current method